tests/e2e/specs/product-editor/block-integration.test.js

```diff
@@ -0,0 +1,693 @@
+/**
+ * External dependencies
+ */
+import { expect, test, Page } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import * as api from '../../utils/api';
+import { getProductBlockEditorUtils } from '../../utils/product-editor';
+
+test.use( { storageState: process.env.ADMINSTATE } );
+test.describe.configure( { mode: 'serial' } );
+
+test.describe( 'Product Block Editor integration', () => {
+	/**
+	 * @type {Page}
+	 */
+	let page = null;
+	let editorUtils = null;
+
+	test.beforeAll( async ( { browser } ) => {
+		page = await browser.newPage();
+		editorUtils = getProductBlockEditorUtils( page );
+
+		await editorUtils.toggleBlockFeature( true );
+		await api.setOnboardedMerchant();
+	} );
+
+	test( 'Prompt to Get Started when not yet finished onboarding', async () => {
+		await api.clearOnboardedMerchant();
+		await editorUtils.gotoAddProductPage();
+		await editorUtils.clickPluginTab();
+
+		const link = page.getByRole( 'link', { name: 'Get Started' } );
+
+		await expect( link ).toBeVisible();
+		await link.click();
```

Comment:
Yes, it can be reduced by about 1 ~ 2 seconds. Adjusted in https://github.com/woocommerce/google-listings-and-ads/pull/2327/commits/caeefeeedff5bb47442cc9ffcb933be98935686b.